src/main/java/dev/snowdrop/release/App.java

```diff
@@ -182,11 +182,10 @@ public void listCVEs(
         final var cves = cveService.listCVEs(Optional.ofNullable(version));
         reportStatus(cves);
         if (release) {
-            final String repoName = "snowdrop/reports";
             if (Optional.ofNullable(token).isPresent()) {
-                git.closeOldCveIssues("cve",token,repoName, version);
+                git.closeOldCveIssues("cve",token, CVEReportingService.CVE_REPORT_REPO_NAME, version);
                 String mdReport = cveReportSvc.buildMdReport(cves,  git.getCveIssueTitle(version));
-                git.createGithubIssue(mdReport, git.getCveIssueTitle(version), "cve", token, repoName);
+                git.createGithubIssue(mdReport, git.getCveIssueTitle(version), "cve", token, CVEReportingService.CVE_REPORT_REPO_NAME);
             } else {
                 LOG.error("Cannot release CVE to GitHub. Github API token is required if --publish is enabled. PLease specify the github token using --token.");
```

Comment:
```suggestion
                LOG.error("Cannot release CVE to GitHub. Github API token is required if --publish is enabled. Please specify the github token using --token.");
```

Reply:
Hmm, that looks weird with the `<span` stuff in there…

Reply:
fixed.